src/main/io/ledstrip.c

```diff
@@ -1276,7 +1276,7 @@ static void applySimpleProfile(timeUs_t currentTimeUs)
 #endif
                 break;
 
-            case LED_PROFILE_BEACON: {
+            case LED_MODE_BEAON: {
```

Comment:
There are a few accidental instances of `BEAON` (without "C")
```suggestion
            case LED_MODE_BEACON: {
```

Reply:
fixed, thanks.
However LED_MODE_COUNT was affected by usage of LED_MODE_ANGLE and LED_MODE_HORIZON, causing compile errors.
So changed these to LED_USER_MODE_xx to differentiate with functional modes.